lib/Driver/ToolChains.cpp

```diff
@@ -124,6 +124,7 @@ static void addCommonFrontendArgs(const ToolChain &TC,
 
   inputArgs.AddAllArgs(arguments, options::OPT_I);
   inputArgs.AddAllArgs(arguments, options::OPT_F);
+  inputArgs.AddAllArgs(arguments, options::OPT_Fsystem);
```

Comment:
The argument order matters here. Should we keep -F and -Fsystem options interleaved, or always put -Fsystem first or last?